src/bp-members/bp-members-functions.php

```diff
@@ -5474,11 +5509,121 @@ function bb_set_bluk_user_profile_slug( $user_ids ) {
 	}
 
 	foreach ( $user_ids as $user_id ) {
-		$user_id           = (int) $user_id;
-		$unique_identifier = bb_generate_user_profile_slug( $user_id );
-		if ( ! empty( $unique_identifier ) ) {
-			bp_update_user_meta( $user_id, 'bb_profile_slug', $unique_identifier );
-			bp_update_user_meta( $user_id, 'bb_profile_slug_' . $unique_identifier, $user_id );
+		bb_set_user_profile_slug( (int) $user_id );
+	}
+}
+
+/**
+ * Function to generate the unique keys.
+ *
+ * @since BuddyBoss [BBVERSION]
+ *
+ * @param int $max_ids How many unique IDs need to be generated? Default 1.
+ *
+ * @return array
+ */
+function bb_generate_uuids( $max_ids = 1 ) {
```

Comment:
The name should be changed to `bb_generate_user_random_profile_slugs` or something similar to this instead of calling generate uuids. because we are not generating a uuids here. it's a unique slugs only.